pkg/resource/interface.go

```diff
@@ -1,4 +1,4 @@
-package reconciler
+package resource
```

Comment:
Why renaming it from `reconciler` to `resource`?
Especially since all instantiations are called `Reconciler`, and comments are still talking about `reconciler`.

Reply:
Because the package is actually pkg/resource :D

Reply:
> Because the package is actually pkg/resource :D

It looks to me that it was not before this PR, and it has been renamed right here in this PR. :thinking: 
![image](https://github.com/user-attachments/assets/dbc2afda-7bc1-4128-82b6-0869bd65491b)

Reply:
I don't remember this discussion with Theo.
And I don't know what are yo services.

The base reason for my question was that:
- the interface changed package and is now `resource` instead of `reconciler`
- but the code and comments are still talking about `reconciler` in lots of places, which can be confusing.

I'm sorry, I feel like I'm saying nonsense and I can't get understood + I can't understand your answers :rofl: 
I'll assume for the best, you know what you're doing even though I don't :sweat_smile:

Reply:
Thanks :)
The discussion was in another PR, sorry. I understand what you mean though. What I want to say is that in the observability operator we have a service/domain layer with the grafana service, Mimir service, bundle configuration service and here we have reconcilers. I plan on moving the logging operator to the same code architecture so we can move services from one operator to the next. That will required removing the reconciler interface anyway.

Now the current code will still talk about réconciliées, and this is not fine so i'll update those next week